packages/opencensus-instrumentation-redis/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+
+{
+  "extends": "./node_modules/gts/tsconfig-google.json",
+  "compilerOptions": {
+    "rootDir": ".",
+    "outDir": "build",
+    "pretty": true,
+    "module": "commonjs",
+    "target": "es6",
+    "strictNullChecks": false
```

Comment:
Although some `opencensus-node` packages have this set to `false`, since this is a new module, could you set to `true` so we get the stronger type checking in the new code?

Could you also set `noUnusedLocals` to `true`?

Reply:
+1 on this.

Reply:
Sure, i believe we could switch them all to strict type checking